### ISSUE TITLE
site_intro_text does not change existing values

### ISSUE THREAD

**Issue by frafra** [ckan/ckan#6540]
**CKAN version** 2.9.4

**Describe the bug**
If I set a different `ckan.site_intro_text`, the text shown in the homepage does not change.

**Steps to reproduce**
1. Set `ckan.site_intro_text`
2. Start CKAN
3. Change`ckan.site_intro_text`
4. Restart CKAN

**Expected behavior**
The new text should be shown.

**Comment by Zharktas** [comment]
I can't reproduce this myself, but did you at any point save the configuration in ckan-admin ? as those are stored in database and database entries always override config variables.

**Comment by frafra** [comment]
You are right. It happens only when I modify the website configuration in the meanwhile, even if I do not intend to change the intro text explicitly (I was changing the theme layout, which does not seem to be possible by using the conf file).

**Comment by Zharktas** [comment]
Config values which are modifiable in ckan-admin are more or less initial values. When you click save in ckan-admin, whatever there is in the form are then used forever.

**Comment by frafra** [comment]
Still, shouldn't a different intro text have some effect and not behave like an initial value?

**Comment by Zharktas** [comment]
@frafra it's the same for all the fields in ckan-admin which get their initial value from config. If you save the settings in ckan-admin , none of those values will be read from config thereafter. This includes logo, site_name, description and probably others.
